### PR TITLE
feat: remove unused exports, functions and files

### PR DESCRIPTION
**Pull request by artemmufazalov**
- Added command, that searches for unused code
- Removed unused code
- Removed unused imports
- Added some imports to `lib.ts` - everything should be imported from there, when app is used as package

There are still unused exports and files, I didn't delete everything

### Test Status: <span style="color: orange;">⚠️ FLAKY</span>
  📊 [Full Report](https://ydb-platform.github.io/ydb-embedded-ui/1750/)

  | Total | Passed | Failed | Flaky | Skipped |
  |:-----:|:------:|:------:|:-----:|:-------:|
  | 222 | 221 | 0 | 1 | 0 |

  😟 No changes in tests. 😕

  ### Bundle Size: 🔽
  Current: 65.84 MB | Main: 65.86 MB
  Diff: 0.01 MB (-0.02%)

  ✅ Bundle size decreased.

  <details>
  <summary>ℹ️ CI Information</summary>

  - Test recordings for failed tests are available in the full report.
  - Bundle size is measured for the entire 'dist' directory.
  - 📊 indicates links to detailed reports.
  - 🔺 indicates increase, 🔽 decrease, and ✅ no change in bundle size.
  </details>

### CODE REVIEW

File: src/containers/Operations/Operations.tsx [hunk 1]
```diff
@@ -62,6 +63,7 @@ export function Operations({database}: OperationsProps) {
                 {data ? (
                     <ResizeableDataTable
                         columns={getColumns({database, refreshTable: refetch})}
+                        columnsWidthLSKey={OPERATIONS_SELECTED_COLUMNS_KEY}
```
Comment (artemmufazalov) [inline src/containers/Operations/Operations.tsx:66]: This property was forgot

File: src/containers/Storage/StorageNodes/PaginatedStorageNodesTable.tsx [hunk 2]
```diff
@@ -3,16 +3,23 @@ import React from 'react';
...
 import {getStorageNodes} from './getNodes';
 import i18n from './i18n';
-import {getRowUnavailableClassName} from './shared';
+
+import './StorageNodes.scss';
+
+const b = cn('ydb-storage-nodes');
+
+const getRowUnavailableClassName = (row: PreparedStorageNode) =>
+    b('node', {unavailable: isUnavailableNode(row)});
```
Comment (artemmufazalov) [inline src/containers/Storage/StorageNodes/PaginatedStorageNodesTable.tsx:22]: Code from `StorageNodes/shared.ts`

File: src/containers/Storage/StorageNodes/PaginatedStorageNodesTable.tsx [hunk 3]
```diff
@@ -3,16 +3,23 @@ import React from 'react';
...
+import {NodesUptimeFilterValues, isUnavailableNode} from '../../../utils/nodes';
 
 import {StorageNodesEmptyDataMessage} from './StorageNodesEmptyDataMessage';
 import {STORAGE_NODES_COLUMNS_WIDTH_LS_KEY} from './columns/constants';
 import type {StorageNodesColumn} from './columns/types';
 import {getStorageNodes} from './getNodes';
 import i18n from './i18n';
-import {getRowUnavailableClassName} from './shared';
+
+import './StorageNodes.scss';
```
Comment (artemmufazalov) [inline src/containers/Storage/StorageNodes/PaginatedStorageNodesTable.tsx:17]: File was not imported, so unavailable nodes wasn't grey

File: src/store/reducers/tooltip.ts [hunk 4]
```diff
@@ -15,7 +15,7 @@ const initialState: ITooltipState = {
     toolTipVisible: false,
     currentHoveredRef: undefined,
     data: undefined,
-    templateType: 'pool',
+    templateType: 'cell',
```
Comment (artemmufazalov) [inline src/store/reducers/tooltip.ts:18]: Our redux tooltip requires some initial value to be set

**Review by Raubzeug — APPROVED** [review]

**Pull request merged**